actionpack/lib/action_dispatch/http/request.rb

```diff
@@ -318,7 +318,11 @@ def GET
     # Override Rack's POST method to support indifferent access
     def POST
       @env["action_dispatch.request.request_parameters"] ||= Utils.deep_munge(normalize_encode_params(super || {}))
-    rescue Rack::Utils::ParameterTypeError, Rack::Utils::InvalidParameterError => e
+    rescue Rack::Utils::ParameterTypeError, Rack::Utils::InvalidParameterError, EOFError => e
+      if e.is_a?(EOFError) && get?
+        return {}
```

Comment:
Because of the workaround for IE here https://github.com/rails/rails/commit/bc254cc23558f016c3697ecd5d39e58f46908018